incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -328,6 +329,19 @@ Similarly, if you want to make `ns-bar` a root again, you must be an
 administrator of the root namespace that is an ancestor of `ns-bar`, since the
 admins of that namespace will lose access to `ns-bar` once it becomes a root.
 
+<a name="resolve-cond"/>
+
+### Resolve conditions on a namespace
+
+If the namespace has the following condition
+```
+ActivitiesHalted (ParentMissing): Parent "parent" does not exist
```

Comment:
Maybe instead of `"parent"` say something like `"<namespace>"`?

Reply:
done